web/src/pages/api/user/login.ts

```diff
@@ -28,19 +28,15 @@ export default async function Login(req: Request, res: NextApiResponse) {
   });
 
   if (!user) {
-    res.statusCode = 400;
-    res.json({
-      success: false,
```

Comment:
I removed all the success sine it is not needed when we are already sending back a status code